xml/System.Text.Json/JsonReaderOptions.xml

```diff
@@ -116,7 +116,7 @@ By default, the reader throws a <xref:System.Text.Json.JsonException> if it enco
 
 ## Remarks
 
-Reading past this depth will throw a <exception cref="T:System.Text.Json.JsonException>.
+Reading past this depth will throw a <xref:System.Text.Json.JsonException>.
```

Comment:
This was not showing up correctly:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonreaderoptions.maxdepth?view=netcore-3.0

![image](https://user-images.githubusercontent.com/6527137/64503324-59283e80-d27f-11e9-88a3-038504470262.png)

Reply:
Yikes, thanks for catching that. They should've been caught with a failing build.

Reply:
There are a bunch of links that the current build system doesn't report as broken. This should improve once we move to SDP. But we'd have to test this scenario when we migrate. It might be that doesn't validate anything because it's expecting Markdown so it just renders the text.